docs/guidance/publish/system_architectures.md

```diff
@@ -1,102 +1,104 @@
 # System architectures
 
-The publication of OCDS data typically requires the design of an architecture that extracts data from live systems, converts it to OCDS format and serves it to users and third party systems.
+The publication of OCDS data involves the creation of a conversion process. Like an ETL process, data has to be extracted from one or more sources, converted and either served or stored.
 
-The design of an architecture depends on several factors:
+This process needs an adequate architecture to support it. Its design depends on several factors:
 
-* Whether existing systems will be modified, or a new system will be built,
+* Whether is possible to extend existing systems, or a new system will be built,
 * The number and nature of the live systems,
-* The technical resources available to the publisher, like storage and processing capabilities, and the availability of technical personnel to maintain systems and modules.
+* The technical resources available to the publisher, like storage and processing capabilities. This includes the availability of technical personnel to maintain systems and modules.
 
-In the ideal scenario:
+Other considerations that affect the design are:
 
-* Individual releases and records for each contracting process should be available at unique persistent URLs;
-* Bulk downloads in JSON, CSV (and, if appropriate, Excel) format should be available covering set periods of contracting;
-* Users should be able to easily locate the collections of releases and records they want.
+* Individual OCDS documents for each process should be available at unique persistent URLs.
+* Bulk downloads in JSON, CSV (and, if appropriate, Excel) formats should be available. These files should be segmented according to one or more criteria, like time periods.
+* Users should be able to locate the collections of releases and records they want.
 
-The following guidance describes some high-level approaches that can be used, with their advantages and disadvantages. These approaches are by no means an extensive or exclusive list, but can be used to inform the design of OCDS publication systems.
+This guidance describes some design approaches with their advantages and disadvantages. This is not an extensive list, but it can be used to inform the design of the publication system.
```

Comment:
```suggestion
This guidance describes some design approaches with their advantages and disadvantages. This is not an exhaustive list, but it can be used to inform the design of the publication system.
```